start-page/engines/base.js

```diff
@@ -7,6 +7,15 @@ eng.duckduckgo = {
 	uri: "https://www.ddg.co/?q=%query%"
 };
 
+/*	Brave Search	
```

Comment:
Brave already exists upstream. Please rebase, or do a pull request in your own branch to import all the upstream commits from feren-os.github.io/master into the :master branch of your fork, and then figure out commits from there.